src/detectors/ZDC/ZDC.cc

```diff
@@ -158,12 +158,12 @@ extern "C" {
             "HcalFarForwardZDCImagingProtoClusters", {"HcalFarForwardZDCSubcellHits"}, {"HcalFarForwardZDCImagingProtoClusters"},
             {
                 .neighbourLayersRange = 1,
-                .localDistXY = {0.76*side_length, 0.76*side_length*sin(M_PI/3)},
-                .layerDistXY = {0.76*side_length, 0.76*side_length*sin(M_PI/3)},
+                .localDistXY = {0.5*side_length, 0.5*side_length*sin(M_PI/3)},
+                .layerDistXY = {0.25*side_length, 0.25*side_length*sin(M_PI/3)},
                 .layerMode=eicrecon::ImagingTopoClusterConfig::ELayerMode::xy,
                 .sectorDist = 10.0 * dd4hep::cm,
                 .minClusterHitEdep = 100.0 * dd4hep::keV,
-                .minClusterCenterEdep = 3.0 * dd4hep::MeV,
+                .minClusterCenterEdep =3.0 * dd4hep::MeV,
```

Comment:
Minor nitpick: please keep spacing consistent
```suggestion
                .minClusterCenterEdep = 3.0 * dd4hep::MeV,
```